text-generation-inference/server/text_generation_server/jetstream_pt_support/engine_loader.py

```diff
@@ -38,9 +50,9 @@ def load_llama_model_info(config: "PretrainedConfig") -> Any:
 
 
 def load_model_info(config: "PretrainedConfig") -> Any:
-    # For now only Llama is supported
-    if config.model_type == "llama":
-        return load_llama_model_info(config)
+    # For now only few models are supported
+    if config.model_type in ["llama", "gemma"]:
+        return load_llama_gemma_model_info(config)
```

Comment:
Aren't we checking this already in `compatibility.py`? Make let's use the same codebase if need to be dupplicated check to avoid tracking down all the model archi we supports in many places moving forward